src/routes.js

```diff
@@ -3,12 +3,13 @@
 /* eslint-disable import/no-named-as-default */
 /* eslint-disable no-unused-vars */
 import React from 'react';
-import {
-  Router, Route as DefaultRoute, Switch, Redirect
-} from 'react-router-dom';
+import { Router, Route as DefaultRoute, Switch, Redirect } from 'react-router-dom';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break after this opening brace.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)